contracts/contracts/OneTimeSBT.sol

```diff
@@ -49,17 +49,17 @@ contract OneTimeSBT is ERC721Enumerable {
         // Convert the last four parameters into a valid timestamp, adding 30 years to adjust for block.timestamp starting in 1970
         uint[6] memory dateNum;
         for (uint i = 0; i < 6; i++) {
-            dateNum[i] = p_proof.pubSignals[PROVE_RSA_COMMITMENT_INDEX + i];
+            dateNum[i] = p_proof.pubSignals[PROVE_RSA_CURRENT_DATE_INDEX + i];
         }
-        uint currentTimestamp = getCurrentTimestamp(dateNum);
+        uint currentTimestamp = _getCurrentTimestamp(dateNum);
```

Comment:
getCurrentTimestamp is used only here, so I made this function as internal and change the name of it in _camelCase